.circleci/config.yml

```diff
@@ -90,20 +93,14 @@ orbs:
     jobs:
       checkout:
```

Comment:
We might want to rename this while we are at it. Definitely confusing reading through the workflow and seeing:

```
requires:
  - checkout
```

Since `checkout` is a built-in step for CircleCI.

Reply:
And... its gone!